### PR TITLE
init project

### PR DESCRIPTION
**Pull request by thaohq-0814**
Init project
Link Redmine: https://edu-redmine.framgia.vn/issues/5122

### CODE REVIEW

**Comment by thaohq-0814** [comment]
ready

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <TextView
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/activity_main.xml]: remove Textview test

**Comment by thaohq-0814** [comment]
ready

File: Music_50.iml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tuanlvt-1117) [inline Music_50.iml]: không push file này lên e

File: Music_502.iml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module external.linked.project.id="Music_502" external.linked.project.path="$MODULE_DIR$" external.root.project.path="$MODULE_DIR$" external.system.id="GRADLE" type="JAVA_MODULE" version="4">
```
Comment (tuanlvt-1117) [inline Music_502.iml]: as above

File: app/app.iml [hunk 4]
```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module external.linked.project.id=":app" external.linked.project.path="$MODULE_DIR$" external.root.project.path="$MODULE_DIR$/.." external.system.id="GRADLE" type="JAVA_MODULE" version="4">
```
Comment (tuanlvt-1117) [inline app/app.iml]: as above

File: app/build.gradle [hunk 5]
```diff
@@ -0,0 +1,28 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "music_50.framgia.com.music_50"
```
Comment (tuanlvt-1117) [inline app/build.gradle]: rename 
```suggestion
        applicationId "com.framgia.music_50"
```

File: app/src/androidTest/java/music_50/framgia/com/music_50/ExampleInstrumentedTest.java [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
+    @Test
+    public void useAppContext() {
+        // Context of the app under test.
+        Context appContext = InstrumentationRegistry.getTargetContext();
+
+        assertEquals("music_50.framgia.com.music_50", appContext.getPackageName());
```
Comment (tuanlvt-1117) [inline app/src/androidTest/java/music_50/framgia/com/music_50/ExampleInstrumentedTest.java]: as above

**Comment by thaohq-0814** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**